Hatching Triage 1.0.1 (#12332)
demisto/content: Packs/HatchingTriage/ReleaseNotes/1_0_1.md

#### Integrations
##### Hatching Triage

> - Added display fields for Integration parameters

Reviewer: ```suggestion
- Added display fields for the Integration parameters.
```

- Updated the Docker image to: *demisto/python3:3.9.4.18682*.

> - Added DBot support to static and behavioural report commands

Reviewer: ```suggestion
- Added DBot support to static and behavioral report commands.
```

> - Fixed error in selecting a Triage profile at submission

Reviewer: ```suggestion
- Fixed an issue where an error occurred when selecting a Triage profile at submission.
```
